maplibre/src/style/style.rs

```diff
@@ -1,10 +1,14 @@
 //! Default vector tile styles configuration.
+// use std::env;
 
-use std::{collections::HashMap, str::FromStr};
+use std::{collections::HashMap, io, path::Path, str::FromStr};
 
-use csscolorparser::Color;
+use csscolorparser::Color as CssColor;
```

Comment:
It would be good if we could limit the dependencies. Is it hard to implement this crate on our own. What does the maplibre spec say about color parsing?